app/src/main/AndroidManifest.xml

```diff
@@ -31,11 +31,11 @@
 
         <activity
             android:name="com.termux.app.TermuxActivity"
-            android:label="@string/application_name"
-            android:configChanges="keyboard|keyboardHidden|orientation|screenSize"
+	    android:label="@string/application_name"
+	    android:configChanges="orientation|screenSize|smallestScreenSize|density|screenLayout|uiMode|keyboard|keyboardHidden|navigation"
```

Comment:
@jeansch: Is `uiMode` necessary to include here? It breaks automatically switching between light and dark theme with #1476. If I remove it, it switches automatically, and the terminals I have open remain open with all their contents.

Reply:
Hi, I don't know, I don't have much knowledge in android development. I just followed documentation link from #1259. 
I don't use the S10 anymore. I would suggest to give it a try, then PR if needed.

Reply:
Ah, so copied from Samsungs documentation, okay. Unfortunately, it doesn't seem like Samsung has documented why it is specified, and I don't have any Samsung devices, so I can't test if it breaks if I remove it.